infra/nightly.sh

```diff
@@ -59,14 +59,32 @@ function perf {
     clean
     xz -d -k -f infra/points.json.xz
     racket -y time.rkt --dir "$REPORTDIR" --profile profile.json infra/points.json
-    python3 infra/ratio_plot.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"/ratio_plot.png
-    python3 infra/point_graph.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"/point_graph.png
-    python3 infra/histograms.py -t "$REPORTDIR"/timeline.json -o1 "$REPORTDIR"/histogram_valid.png -o2 "$REPORTDIR"/histogram_all.png
+    python3 infra/ratio_plot.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"
+    python3 infra/point_graph.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"
+    python3 infra/histograms.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"
+    python3 infra/cnt_per_iters_plot.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"
+    python3 infra/repeats_plot.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"
+    python3 infra/density_plot.py -t "$REPORTDIR"/timeline.json -o "$REPORTDIR"
     cp profile.json "$REPORTDIR"/profile.json
     cp profile.js "$REPORTDIR"/profile.js
 }
 
-for cmd in $@; do
-    echo "Running $cmd"
-    $cmd
-done
+function bench {
```

Comment:
this function allows to run a particular experiment with making the whole report page
`bash infra/nightly.sh bench [id]`